lib/alarms.ts

```diff
@@ -49,7 +49,7 @@ export default class AlarmsResource {
             alarmDescription:
                 'Alarm when the Stream errors is more than 1 unit for 15 minutes out of the past 25 minutes. Streams do have retry logic',
             alarmName: `FhirSolution.${stage}.High.DDBToESLambdaErrorAlarm`,
-            actionsEnabled: false,
+            actionsEnabled: true,
```

Comment:
Do we need to do anything else like assigning actions or is just setting it to true sufficient?

Reply:
There was already actions added https://github.com/awslabs/fhir-works-on-aws-deployment/blob/faf2a5eecf7e78fac6f90c1f443c0c6bb7c76564/lib/alarms.ts#L68 , but they were never enabled

Reply:
Oops 😅